### PR TITLE
Chord Diagram | Adding forced highlighting to config

### PR DESCRIPTION
**Pull request by reb-dev**
https://github.com/f5/unovis/assets/755708/807a1df1-e274-4798-8135-d7609ff43e30

### CODE REVIEW

File: packages/ts/src/components/chord-diagram/style.ts [hunk 1]
```diff
@@ -78,14 +74,10 @@ export const link = css`
...
-  &:hover {
-    fill-opacity: 1;
-  }
 `
 
-export const hoveredLink = css`
-  label: hovered;
+export const highlightedLink = css`
+  label: highlighted;
   fill-opacity: 0.9;
```
Comment (rokotyan) [inline packages/ts/src/components/chord-diagram/style.ts:81]: I think we also need to add CSS variables to control the opacity of highlighted / dimmed nodes and links

**Review by rokotyan — COMMENTED** [review]
Works well! 

The only problem I see is that the highlighted links are a bit difficult to distinguish from the dimmed ones. Let's tweak their opacity a little bit, and add CSS variables for them.

Also the pipeline is failing

**Comment by reb-dev** [comment]
@rokotyan Fixed the pipeline issue and added the CSS variables mentioned above.

**Comment by rokotyan** [comment]
@reb-dev Thanks, merging

**Review by rokotyan — APPROVED** [review]

**Pull request merged**